packages/terra-application-utility/src/utility/_UtilityMenuDivider.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './_UtilityMenuDivider.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * If the divider shall be placed before the footer
+   */
+  isFooter: PropTypes.bool,
```

Comment:
This would be more reusable if you go with isBottom. Footer seems to imply 1 usage.

Reply:
Divider changes [here](https://github.com/cerner/terra-framework/pull/103/commits/efd64c3eed5f7146aaefa0505339e34e92f75bed#diff-4ef53a760f640de1e54b07fae855394dR10).